providers/base/units/stress/jobs.pxu

```diff
@@ -607,19 +607,26 @@ plugin:shell
 category_id: com.canonical.plainbox::stress
 id: stress/s2idle_pm-graph_30
 estimated_duration: 10m
-requires: 
+requires:
+ cpuinfo.type == 'GenuineIntel'
  executable.name == 'sleepgraph'
  sleep.mem_sleep == 's2idle'
 user: root
 _summary: Resume from idle by using Intel pm-graph
 command:
- sleepgraph -m freeze -rtcwake 10 -sync -gzip -multi 30 0 -skiphtml -o "$PLAINBOX_SESSION_SHARE"/s2idle_pm-graph/s2idle-"$(date -d today +%Y-%m-%d-%H%M)"
+ if [ ! -f /proc/driver/nvidia/suspend ]; then
+     sleepgraph -m freeze -rtcwake 60 -sync -gzip -multi 30 30 -skiphtml -o "$PLAINBOX_SESSION_SHARE"/s2idle_pm-graph/s2idle-"$(date -d today +%Y-%m-%d-%H%M)"
+ else
+     echo "The platform has Nvidia driver loaded, which is not capable of running sleepgraph"
+     exit 1
+ fi
 
 plugin: attachment
```

Comment:
Have you tried running this job with the modification you made to `stress/s2idle_pm-graph_30` on a device with a Nvidia GPU? `stress/s2idle_pm-graph_30` will not run sleepgraph, so nothing will be generated and the attachment job will complain it doesn't find the directory you're talking about.

You probably need to modify the `command` to check for the existence of `"$PLAINBOX_SESSION_SHARE"/s2idle_pm-graph` first.